<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlyUI</class>
 <widget class="QWidget" name="FlyUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>939</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Flight Controller</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="vehicleParameterButton">
        <property name="text">
         <string>Vehicle Parameters</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Advanced</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPushButton" name="pollENUrefButton">
        <property name="text">
         <string>Poll ENU ref. from Vehicle</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="armButton">
     <property name="text">
      <string>Arm</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="disarmButton">
     <property name="text">
      <string>Disarm</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="takeoffButton">
     <property name="text">
      <string>Takeoff</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="returnToHomeButton">
     <property name="text">
      <string>Return to Home</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="landButton">
     <property name="text">
      <string>Land Here</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="precisionLandButton">
     <property name="text">
      <string>Precision Land</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="gotoButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>123</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Go to:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="getPositionButton">
          <property name="text">
           <string>Get Pos.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="gotoXSpinBox">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="prefix">
         <string>x: </string>
        </property>
        <property name="suffix">
         <string>m</string>
        </property>
        <property name="minimum">
         <double>-9999.989999999999782</double>
        </property>
        <property name="maximum">
         <double>9999.989999999999782</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="gotoYSpinBox">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="prefix">
         <string>y: </string>
        </property>
        <property name="suffix">
         <string>m</string>
        </property>
        <property name="minimum">
         <double>-9999.989999999999782</double>
        </property>
        <property name="maximum">
         <double>9999.989999999999782</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="gotoZSpinBox">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="prefix">
         <string>z: </string>
        </property>
        <property name="suffix">
         <string>m</string>
        </property>
        <property name="maximum">
         <double>999.990000000000009</double>
        </property>
        <property name="value">
         <double>2.500000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="autopilotGroupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Autopilot</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QRadioButton" name="apExecuteRouteRadioButton">
          <property name="text">
           <string>Execute Route</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="apFollowPointRadioButton">
          <property name="text">
           <string>Follow Point</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="apRestartButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../resources.qrc">
            <normaloff>:/icons/icons/Restart-96.png</normaloff>:/icons/icons/Restart-96.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="apStartButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../resources.qrc">
            <normaloff>:/icons/icons/Play-96.png</normaloff>:/icons/icons/Play-96.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="apPauseButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../resources.qrc">
            <normaloff>:/icons/icons/Pause-96.png</normaloff>:/icons/icons/Pause-96.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="apStopButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../resources.qrc">
            <normaloff>:/icons/icons/Stop-96.png</normaloff>:/icons/icons/Stop-96.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="vehicleIdToFollowLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Follow Vehicle ID:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="followVehicleIdCombo"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
